.github/workflows/publish-extensions.yml

```diff
@@ -17,14 +19,14 @@ jobs:
   publish_chrome_extension:
     name: Publish Chrome extension
     runs-on: ubuntu-latest
-    if: github.event.head_commit.message == 'Version release'
+    if: startsWith(github.ref, 'refs/tags/v')
```

Comment:
This along with the firefox publish step below are the only two places where I can remove dependencies on the git commit message. The dependency is still in a couple other places and likely can't be removed without fully automating releases.

Reply:
This is totally fine -- and actually exactly what we want!